glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
Thanks for pointing out.
But how to create other namespaces without starting any additional process?
Must be there some ways like network, but I can't find out any grace way currently.

------------
I found a bad way (untested)
the creating process()
  unshare()
  mount --bind to save the namespace on some where.
  setns() back or exit.

the starting init process()
  setns() the the save namespace.
-----
So I think it is better to create namespaces when starting the container.